roles/copy_container/molecule/default/converge.yml

```diff
@@ -0,0 +1,42 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  tasks:
+    - name: Run copy-container role
+      ansible.builtin.include_role:
+        name: copy_container
+
+    - name: Deploy Local Registry
+      ansible.builtin.include_role:
+        name: registry_deploy
+
+    - name: Copy containers from RDO quay to local registry
+      ansible.builtin.command:
+        cmd: >-
+          copy-quay --debug --pull-registry quay.rdoproject.org
+          --from-namespace podified-antelope-centos9
+          --push-registry 127.0.0.1:5001
+          --to-namespace podified-antelope-centos9 --release master
+          copy openstack-os
+      args:
+        chdir: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/roles/copy_container/files/copy-quay"
```

Comment:
Hi @arxcruz We are trying to add molecule job to test copy_container role.
At this step in the molecule job, we are hitting [following issue ](https://logserver.rdoproject.org/02/1002/ea75b0c27af90efd6604c3f6c6b7fec5ac234298/github-check/cifmw-molecule-copy_container/0384143/job-output.txt)while passing release as antelope or master
```
 TASK [Copy containers from RDO quay to local registry] *************************
2024-01-05 05:11:04.144870 | controller | fatal: [instance]: FAILED! => changed=true
2024-01-05 05:11:04.144952 | controller |   cmd:
2024-01-05 05:11:04.144961 | controller |   - copy-quay
2024-01-05 05:11:04.144976 | controller |   - --debug
2024-01-05 05:11:04.144983 | controller |   - --pull-registry
2024-01-05 05:11:04.145443 | controller |   - quay.rdoproject.org
2024-01-05 05:11:04.145490 | controller |   - --from-namespace
2024-01-05 05:11:04.145502 | controller |   - podified-antelope-centos9
2024-01-05 05:11:04.145522 | controller |   - --push-registry
2024-01-05 05:11:04.145536 | controller |   - 127.0.0.1:5001
2024-01-05 05:11:04.145744 | controller |   - --to-namespace
2024-01-05 05:11:04.146294 | controller |   - podified-antelope-centos9
2024-01-05 05:11:04.146358 | controller |   - --release
2024-01-05 05:11:04.146370 | controller |   - master
2024-01-05 05:11:04.146380 | controller |   - copy
2024-01-05 05:11:04.146397 | controller |   - openstack-os
2024-01-05 05:11:04.146481 | controller |   delta: '0:00:00.033781'
2024-01-05 05:11:04.146644 | controller |   end: '2024-01-05 05:11:04.103971'
2024-01-05 05:11:04.146675 | controller |   msg: non-zero return code
2024-01-05 05:11:04.147069 | controller |   rc: 1
2024-01-05 05:11:04.147096 | controller |   start: '2024-01-05 05:11:04.070190'
2024-01-05 05:11:04.147107 | controller |   stderr: time="2024-01-05T05:11:04-05:00" level=fatal msg="no entry found for release master"
2024-01-05 05:11:04.147646 | controller |   stderr_lines: <omitted>
2024-01-05 05:11:04.147670 | controller |   stdout: ''
2024-01-05 05:11:04.147681 | controller |   stdout_lines: <omitted>
```
Please have a look, thank you :-)